articles/quickstart/spa/angular-next/02-calling-an-api.md

```diff
@@ -0,0 +1,186 @@
+---
+title: Call an API
+description: This tutorial demonstrates how to make API calls for protected resources on your server.
+budicon: 448
+topics:
+  - quickstarts
+  - spa
+  - angular
+  - api
+github:
+  path: Sample-01
+contentType: tutorial
+useCase: quickstart
+---
+<!-- markdownlint-disable MD002 MD034 MD041 -->
+
+When you use Auth0, you delegate the authentication process to a centralized service. Auth0 provides you with functionality to log in and log out users from your Angular application. However, your application may need to access resources from a secured API.
+
+You can also [protect an API with Auth0](https://auth0.com/docs/microsites/protect-api/protect-api). We offer multiple [Auth0 API quickstarts](https://auth0.com/docs/quickstart/backend) to help you integrate Auth0 with your backend platform.
+
+When you use Auth0 to protect your API, you delegate the authentication process to a centralized service that ensures only approved applications can access private resources. Your Angular application authenticates the user and receives an [access token from Auth0](https://auth0.com/docs/tokens/concepts/access-tokens). The application can then pass that access token to your API as a credential. In turn, your API can use [Auth0 libraries](https://auth0.com/docs/libraries) to verify the access tokens it receives from the calling application and issue a response with the desired data.
```

Comment:
I would suggest rephrasing so that the description applies to any API protected by Auth0, as opposed to the current formulation that refers only to the custom API. Example (also note small tweaks like authn->authz):

When an API is protected by Auth0, the authorization process is delegated to a centralized service that ensures only approved applications can access private resources. Your Angular application authenticates the user and, in the process, it also receives an access token from Auth0. The application can then pass that access token to the API as a credential, as prescribed by OAuth2. In turn, the API can use Auth0 libraries (or any libraries capable to process JWT, the open standard Auth0 uses to encode access tokens) to verify the access tokens it receives from the calling application and issue a response with the desired data.

Reply:
Thanks @vibronet. Yes this was mostly lifted from the React article as I believed that to be mostly correct, aside from the mention of Angular. You're right @lbalmaceda in that we could abstract this out and use in both places 👍